lab_1/main.py

```diff
@@ -36,7 +55,15 @@ def calculate_frequencies(tokens: list) -> dict:
     e.g. tokens = ['weather', 'sunny', 'man', 'happy']
     --> {'weather': 1, 'sunny': 1, 'man': 1, 'happy': 1}
     """
-    pass
+    freq_dict = {}
+    try:
```

Comment:
`try ... except` blocks are usually wrap the line where the Exception is likely to be raised.

Take a look at `isinstance` for type-checking.